aspnetcore/security/authorization/secure-data/samples/final15/Authorization/ContactRoleAuthorizationHandler.cs

```diff
@@ -16,7 +16,7 @@ public class ContactRoleAuthorizationHandler
                 return Task.FromResult(0);
             }
 
-            if (context.User.IsInRole(requirement.Name))
+            if (context.User.IsInRole(Constants.OperationRolePrefix + requirement.Name))
```

Comment:
@Rick-Anderson I'm making progress updating the sample per @blowdart 

Since we need to seed 2 users, an administrator and another user who can create contacts, I was thinking of making the user names admin@contoso.com and user@contoso.com. Then any user created via the registration ui will be only guest with read permission, so only authenticated users can access the contacts at all. The admin use will be able to do anything and the normal user will be able to view all contacts, create contacts and edit/delete their own contacts.
My questions for you are:

** Do those name changes sound ok to you?
** Was thinking for sample purposes maybe the seeded users should be listed on the home page for easy discovery?
** Do we want to require the creation of 2 user secrets, one password for each user? or use the same password for both?
** My own opinion is that for sample purposes requiring user secrets to be configured before using the sample adds a bit of friction. I would just hard code the passwords and even list them with the user names on the home page so there is no setup required other than building the solution and instructions could be on the home page telling about the pre-configured sample users and what they are allowed to do, perhaps with some caveat about this is only a sample users should not be hard coded in real world apps. But I defer to whatever you suggest.
** The sample has a class ContactHasOneAuthorizationHandler (file name is actually ContactNotOneAuthorizationHandler) which is not being used in the contacts controller. Should we keep that or remove it or at least make the name file name and class name the same?